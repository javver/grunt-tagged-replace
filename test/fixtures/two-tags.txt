two tags1 /*tag1*/ /*/tag1*/ end two tags2
two tags1 /*tag2*/  to be replaced /*/tag*2/ end two tags2
