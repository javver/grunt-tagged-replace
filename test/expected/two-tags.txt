two tags1 /*tag1*/ replaced content 1 /*/tag1*/ end two tags2
two tags1 /*tag2*/ replaced content 2 /*/tag*2/ end two tags2
